meduzzen_backend/meduzzen_backend/settings.py

```diff
@@ -31,17 +31,20 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY')
+SECRET_KEY = os.environ.get('SECRET_KEY', 'change_me')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get('DEBUG')
+DEBUG = bool(int(os.environ.get('DEBUG', 1)))
```

Comment:
why do you int?

Reply:
Can't convert False and True values from .env to boolean. Easy to set 0 or 1